src/themes/teams-dark/components/Button/buttonStyles.ts

```diff
@@ -0,0 +1,164 @@
+import { pxToRem } from '../../../../lib'
+import { IComponentPartStylesInput, ICSSInJSStyle } from '../../../../../types/theme'
+import { IButtonProps } from '../../../../components/Button/Button'
+
+const buttonStyles: IComponentPartStylesInput<IButtonProps, any> = {
```

Comment:
Can we do this without rewriting the entire component styles file? I think for everything here, we want the button to be the same, except for colors. If we do this for every theme, that means if we want to update buttons border-radius, we will need to do that in multiple places.

I think ideally for most things, we would just change the color variables in siteVariables for dark or contrast or any theme and we wouldn't have to re-write any component styles or variables files. Maybe the theme could know that it has a parent theme?